packages/sdk/package.json

```diff
@@ -70,6 +70,7 @@
     "new-github-issue-url": "0.2.1",
     "node-fetch": "2.6.7",
     "open": "7",
+    "ora": "^5",
```

Comment:
Could you replace by the exact version? Renovate will manage the updates after

Reply:
Sure, I'll stick with the latest "5" version (as version 6 and above require ESM)